README.md

```diff
@@ -1,62 +1,94 @@
-# Block Template
+# Block template
 
-The Block Template is a base template used to create a new block type with the recommended files and file structure.
+The block template is a base template used to create a new block type with the recommended files and file structure.
 
-## How to Use
+---
 
-If you need to first create a project directory, the `project_template` repository at [https://github.com/niolabs/project_template](https://github.com/niolabs/project_template) provides a good starting point.
+## How to use
 
-### Clone the Block Template
```

Comment:
I just found this in the docs, maybe emphasize the CLI first in the readme as well? https://docs.n.io/cli/newblock.html

Reply:
The newblock command pretty much automatically does all this stuff below, right?